.github/workflows/benchmark.yml

```diff
@@ -15,10 +15,8 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '*'
-          cache: 'npm'
-          cache-dependency-path: 'npm-shrinkwrap.json'
-          check-latest: true
+          node-version: lts/*
+          cache: npm
```

Comment:
All non-test workflows I changed to use the latest lts version instead of the latest version and not do extra requests to check for the latest. The nodejs version here does not really matter, and I think it is good to use the most stable one which is lts.